### DISCUSSION TITLE
TorrentGalaxy无法添加

### DISCUSSION THREAD

**Discussion by fireflycao** [jxxghp/MoviePilot#2805] [问题咨询]
tgx.rs
torrentgalaxy.mx
用的是这个两个网址

**Comment by charys117** [comment]
> tgx.rs torrentgalaxy.mx 用的是这个两个网址

torrentgalaxy.to这个能过验证

**Reply by hulinhao233** [reply]
用tgx.rs能保存，测试也成功，浏览里刷不出东西
torrentgalaxy.to和torrentgalaxy.mx都直接测试不通过

**Comment by Mingyang0624** [comment]
现在有添加成功吗，我试了上面这几个都不太行诶，测试能成功，但是浏览里面刷不出东西
